### PR TITLE
Add BSON Binary Subtype 7

### PR DESCRIPTION
**Pull request by nspektor**

### CODE REVIEW

File: .swift-version [hunk 1]
```diff
@@ -1 +1 @@
-5.2.4
+5.5.1
```
Comment (nspektor) [inline .swift-version:1]: Should this change be part of the PR? or should I somehow only do it locally? (It was required for tests to pass)

**Review by nspektor — COMMENTED** [review]
Is there anything else that needs to be done to support this new column subtype?

File: .swift-version (hunk 1 above)
Comment (isabelatkinson) [reply .swift-version:1]: hm I'm not sure if we want to bump the version here, @kmahar or @patrickfreed could probably give a better answer

Comment (kmahar) [reply .swift-version:1]: yeah I think it's fine to update this now

**Review by kmahar — APPROVED** [review]
lgtm!

**Review by isabelatkinson — APPROVED** [review]
lgtm!

File: .swift-version (hunk 1 above)
Comment (nspektor) [reply .swift-version:1]: Great, thanks

**Pull request merged**